lib/cambiatus_web/schema/commune_types.ex

```diff
@@ -119,6 +119,18 @@ defmodule CambiatusWeb.Schema.CommuneTypes do
         {:ok, transfer}
       end)
     end
+
+    @desc "Subscribe to highlighted_news change"
+    field :highlighted_news_change, :news do
```

Comment:
we don't need the word `change`

```suggestion
    field :highlighted_news, :news do
```